apps/web/components/Embed.tsx

```diff
@@ -402,6 +402,7 @@ const embeds = (t: TFunction) =>
         title: t("inline_embed"),
         subtitle: t("load_inline_content"),
         type: "inline",
+        disableForRoutingForms: false,
```

Comment:
How about using a generic name just "disabled" and default false?

Reply:
Agreed with @alannnc

Reply:
`disabled` doesn't convey who it's disabled for. 
If you would see down below, we simply doesn't check and return null. We check that it should be routing form context, and then honour disabled field. 

So, it makes more sense to have some context with disabled.

Reply:
@alannnc Sorted it. disabled property isn't needed now, I am filtering out the email type embed in RoutingFormEmbed component